pages/linux/httpie.md

```diff
@@ -0,0 +1,31 @@
+# httpie
+
+> A human-friendly CLI HTTP tool.
+
+- GET method (default method with no request data):
+
+`http {{https://api.github.com/users}}`
+
+- POST method (default method with request data):
+
+`http {{example.org}} {{hello=World}}`
+
+- POST method with redirected input:
+
+`http {{example.org}} < {{file.json}}`
+
+- PUT method:
```

Comment:
Perhaps `Send a PUT request with the specified body` would read better?

Reply:
I think just `Send a PUT request` is OK